src/app/components/layout/MultiSelector.js

```diff
@@ -282,9 +282,8 @@ class MultiSelector extends React.Component {
           }
           <ButtonMain
             className="int-multiselector__button--save"
-            label={this.props.submitLabel}
```

Comment:
@brianfleming here's the missing label :sweat_smile:

Reply:
ohhh! well I am to blame either way! Thanks will put up fix